commands/createNewProject.js

```diff
@@ -26,14 +27,69 @@ const copyAssetsContent = async includeCypress => {
 
     if (!includeCypress) {
       await removeCypressFromPackage();
+      console.log('Cypress dependencies removed!');
     }
 
+    await setMiddleware(middleware);
+    console.log('Middleware files ready!');
     console.log('Copying finished!');
   } catch (err) {
     console.error(err);
   }
 };
 
+const setMiddleware = async middleware => {
+  try {
+    const packageJson = JSON.parse(
+      fs.readFileSync(path.join(paths.outputPath, 'package.json')),
+    );
+
+    let middleWareDeps = {
+      package: middleware === 'Redux Saga' ? 'redux-observable' : 'redux-saga',
```

Comment:
'REdux Saga'as const and reused

Reply:
Such boilerplates are now moved to main config file.